VideoWeb/VideoWeb/ClientApp/package.json

```diff
@@ -53,15 +53,19 @@
         "chart.js": "^2.9.4",
         "classlist.js": "^1.1.20150312",
         "core-js": "^3.8.2",
+        "fsevents": "^2.1.3",
```

Comment:
not required to run the app nor is it needed for non-Mac OS platforms. This can move to the optional section